adodbapi/examples/db_print.py

```diff
@@ -59,10 +59,10 @@
         print("str() of first five records are...")
 
         # get the results
-        db = c.fetchmany(5)
+        db_result = c.fetchmany(5)
 
         # print them
-        for rec in db:
+        for rec in db_result or ():

```

Comment:
Alternatively, `Cursor._fetch` or `Cursor.fetchmany` could be made to return an empty iterable (`[]` or `()`) when `Cursor.connection` or `cursor.rs` is `None`. Instead of the current behaviour of returning `None`. But that would be a change in behaviour.